Cargo.toml

```diff
@@ -27,8 +28,13 @@ quickcheck = { version = "0.9.2", optional = true }
 rand = { version = "0.7.3", optional = true }
 serde = { version = "1.0.116", default-features = false, optional = true }
 serde_bytes = { version = "0.11.5", optional = true }
+arbitrary = { version = "1.1.0", optional = true , features = ["derive"] }
```

Comment:
Nit: other dependencies have the `optional` as last parameter, to make it for humans easy to spot which are optional.